config/jobs/kubernetes-sigs/image-builder/OWNERS

```diff
@@ -0,0 +1,6 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - image-builder-maintainers
```

Comment:
This should be expanded out or an additional OWNERS_ALIASES file added to expand this to individual users.